### DISCUSSION TITLE
Vercel DNS issue - DNS_PROBE_FINISHED_NXDOMAIN

### DISCUSSION THREAD

**Discussion by Odyseawebsite** [vercel/community#3691] [Help]
### Summary

I have connected Vercel (hosting) to my domain, odysea.sg (GoDaddy) via nameservers and everything works fine and the website is up but when I leave it for about 1 hour or so, it automatically disconnects and says the site cannot be reached because DNS_PROBE_FINISHED_NXDOMAIN.

I have connected the nameservers properly but after that 1 hour or so of connection,  my Vercel dashboard says that the configuration is invalid and the nameserver on domain is not found...

Could it be because I need to purchase a paid Vercel plan to host a website to a paid domain? Or could it be a '.sg' domain verification issue? I'm a newbie here I apologise for having this question but can anyone please help me!

The below screenshot shows 'valid configuration' only after I disconnected nameserver and retyped it on GoDaddy. But after an hour, it turns into invalid configuration and and main nameserver not found :(

### Steps to Reproduce

<img width="931" alt="Screen Shot 2023-08-21 at 1 28 11 PM" src="https://github.com/vercel/community/assets/140012496/039af038-4c8b-4aa4-aeb7-68ae87a39801">

**Comment by amyegan** [comment]
Hey @Odyseawebsite. It looks like the domain is working now. Do you still see the `DNS_PROBE_FINISHED_NXDOMAIN` error?
